### PR TITLE
Update Dockerfile

### PR DESCRIPTION
**Pull request by Irio**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,23 +1,22 @@
-FROM python:3.5
+FROM python:3.6.2-slim
```
Comment (cuducos) [inline Dockerfile]: Just out of curiosity: any reason to pick up `slim` in favor of `alpine`? Testing this PR locally and gonna merge if it works properly. Just asking about the distro out of curiosity ; )

File: Dockerfile [hunk 2]
```diff
@@ -1,23 +1,22 @@
...
+COPY config.ini.example ./
 COPY requirements.txt ./
 COPY setup ./
-COPY rosie.py ./
-COPY rosie ./rosie
-COPY config.ini.example ./
-
 RUN ./setup
 
-CMD python rosie.py run
```
Comment (cuducos) [inline Dockerfile]: Running `docker run --rm -v /tmp/serenata-data:/tmp/serenata-data rosie` as stated in the docs doesn't produce anything. I guess this `CMD` line might have been deleted by accident (or new instructions should be provided).

File: Dockerfile (hunk 1 above)
Comment (caduvieira) [reply Dockerfile]: Slim is based on debian

Comment (cuducos) [reply Dockerfile]: And why being based on Bebian is better/desired/required, @caduvieira? Just so we don't have to find equivalent packages in `apk`?

**Comment by cuducos** [comment]
Closed in favor of #92

**Pull request closed without merging**